tests/test_views/admin/instance/test_instance_view.py

```diff
@@ -50,48 +41,33 @@ def test_admin_instance_new_support(self):
         self.assertEquals(302, response.status_code)
         self.assertEquals(number_support_before + 1, Support.objects.all().count())
 
-    def test_customer_cannot_create_new_support_from_admin(self):
```

Comment:
this should be now --> test_customer_cannot_create_support_on_non_owned_instance